apps/web/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Unreleased
 
+## Changes
+
 - Display `requested amount` and `recipient` fields inside Community Pool Spend proposal details page ([\#1053](https://github.com/forbole/big-dipper-2.0-cosmos/pull/1053))
+- Added Connect Wallet pop up screen ([\#1068](https://github.com/forbole/big-dipper-2.0-cosmos/pull/1068))
```

Comment:
Hi @MonikaCat, I propose to use `changesets` to handle CHANGE LOG from now on

https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md

for example, the following change will be created.
https://app.warp.dev/block/N26x633kwhHnNUWQbqEVnx
https://app.warp.dev/block/irqexovjFIzakX0xvgnNkF
and then the changes will be committed
when we merge change into `main`
https://github.com/forbole/big-dipper-2.0-cosmos/blob/88607098d57652cf12a12f7f4704f557c4a2e77d/.github/workflows/release.yml#L30
we have a GItHub action to append change to CHANGE.md

What do you think?

Reply:
Ok nice! I've just updated it, can you review it again pls? Thanks :D

Reply:
@teamchong does it apply to my PR #1085 as well ? Whenever there's a feature to be added, I'd add the commit in CHANGE LOG inside the `.changeset/` ?

Reply:
@rachelhox no only use changeset for big-dipper repo for now